src/components/onboarding/PaymentForm.tsx

```diff
@@ -1,10 +1,19 @@
 import { useCreatePaymentMethod } from "@3rdweb-sdk/react/hooks/useApi";
-import { Center, Flex, Spinner } from "@chakra-ui/react";
+import {
+  Alert,
+  AlertDescription,
+  AlertIcon,
+  Center,
+  Flex,
+  Spinner,
+} from "@chakra-ui/react";
 import {
   useElements,
   useStripe,
   PaymentElement,
 } from "@stripe/react-stripe-js";
+import { PaymentVerificationFailureAlert } from "components/settings/Account/Billing/alerts/PaymentVerificationFailureAlert";
+import message from "components/settings/ApiKeys/message";
```

Comment:
is this used?

also this import makes it look like we're using `export default` inside this file we're importing which we should not do except for extraordinary circumstances

Reply:
Updated, good catch. I didn't realize this was importing a new field for message. Will keep this in mind when reviewing future PRs